src/js/edu-benefits/reducers/veteran/index.js

```diff
@@ -1,14 +1,27 @@
 import _ from 'lodash/fp';
 
 import { VETERAN_FIELD_UPDATE, ENSURE_FIELDS_INITIALIZED } from '../../actions';
-import { makeField } from '../../../common/model/fields';
+import { makeField, dirtyAllFields } from '../../../common/model/fields';
 
 const blankVeteran = {
   benefitsRelinquished: makeField(''),
   chapter30: false,
   chapter1606: false,
   chapter32: false,
-  chapter33: false
+  chapter33: false,
+  veteranFullName: {
```

Comment:
Created this ticket in the JSON schema repo for discussion: https://github.com/department-of-veterans-affairs/vets-json-schema/issues/11